db/migrate/20171118114518_create_delayed_jobs.rb

```diff
@@ -0,0 +1,22 @@
+class CreateDelayedJobs < ActiveRecord::Migration[5.1]
+  def self.up
+    create_table :delayed_jobs, force: true do |table|
```

Comment:
Since you are using `t` in the other migration, I would keep it consistent here as well